health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FormulaConfiguration.js

```
@@ -0,0 +1,106 @@
+              //key={field.key}
+              mainClassName={"checkboxOptionVariant"}          
+              //disabled={optionDependency ? true: false}
+              label={t("Show on Estimation Dashboard")}
+              //checked={optionComment}
+             // onChange={(event) => handleOptionComment(optionId)}
+              isLabelFirst={false}
+              //index={field.key}
+
+              
+              />
+                        <div style={{background:"#eee", height:"0.2rem", marginBottom:"1rem"}}></div>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)